### PR TITLE
feat: nixbuildnet trait

### PR DESCRIPTION
**Pull request by renebarbosafl**
## Summary by CodeRabbit

- **New Features**
	- Integrated the capability to connect to Nixbuild.net for enhanced build network functionality.

### CODE REVIEW

**Comment by renebarbosafl** [comment]
atlantis plan -d infra/us-east-1/dev/eks/kubevela-addons

**Comment by renebarbosafl** [comment]
atlantis apply -d infra/us-east-1/dev/eks/kubevela-addons

**Comment by renebarbosafl** [comment]
atlantis apply -d infra/us-east-1/dev/eks/kubevela-addons

**Comment by renebarbosafl** [comment]
atlantis plan -d infra/us-east-1/dev/eks/kubevela-addons

**Comment by renebarbosafl** [comment]
atlantis apply -d infra/us-east-1/dev/eks/kubevela-addons

File: infra/modules/kubevela-addons/definitions/nixbuildnet.cue [hunk 1]
```diff
@@ -0,0 +1,80 @@
+parameter: {
+    secretName: string
```
Comment (shlevy) [inline infra/modules/kubevela-addons/definitions/nixbuildnet.cue:2]: This is unused. How do we get the `token`?

Comment (renebarbosafl) [reply infra/modules/kubevela-addons/definitions/nixbuildnet.cue:2]: It's still a WIP, @shlevy ! Calm down! 😆 
I had to open a pull request to interact with Atlantis. Marking it as a draft.

**Pull request closed without merging**